code/game/gamemodes/storyteller_print.dm

```diff
@@ -106,32 +115,47 @@
 
 	data += "<hr><b>Current antags:</b><div style=\"border:1px solid black;\"><ul>"
 
-	for(var/datum/antagonist/A in current_antags)
-		var/act = "<font color=red>DEAD</font>"
-		if(!A.is_dead())
-			if(!A.is_active())
-				act = "<font color=silver>AFK</font>"
-			else
-				act = "OK"
-
-		data += "<li>[A.role_text] - [A.owner?(A.owner.name):"no owner"] ([act])<a href='?src=\ref[A];panel=1'>\[EDIT\]</a></li>"
+	if (current_antags.len)
```

Comment:
I think this check does nothing in this case, `for` should work fine on empty lists.